autoyast/lib/main/ay_lib.sh

```diff
@@ -645,17 +671,27 @@ function parse_line()
 
         # field 02 cidr
         my_preflen_1=$(split_ip_cidr "$(print_col 2)" cidr)
+ 
+
         if [ -z "$my_preflen_1" ];then
-                my_preflen_1=$(get_dhcp_info NETMASK)
-                my_ipaddress=$(get_ip_list)
+
+		#JH 2019/08/10
+		#Avoid using get_dhcp_info as it requires a dhcp server to be available
+                #my_preflen_1=$(get_dhcp_info PREFIXLEN)
+                #my_ipaddress=$(get_ip_list)
+		my_ip_and_prefix=$(getIpAddressAndPrefix)
+		my_preflen_1=$(split_ip_cidr "$my_ip_and_prefix" cidr)                
+		my_ipaddress=$(split_ip_cidr "$my_ip_and_prefix" ip)		
+
         fi
 
         # field 03 gateway
         my_gateway="$(print_col 3)"
 	if [ -n "$my_gateway" ];then
 		GATEWAY=$my_gateway
 	else
-		my_gateway=$GATEWAY
+		mo_gateway=$GATEWAY
+t
```

Comment:
Seems we had a typo, here! This was causing the gateway not to be set anymore.. Fixed with next checkin.